### DISCUSSION TITLE
Problem with drone drifing while using Optitrack

### DISCUSSION THREAD

**Discussion by skovsen** [USC-ACTLab/crazyswarm#727] [General]
Hi. I have scoured through the issues, discussion and so forth, but I havent been able to find a solution to my problem yet. 
I have done a couple of project with a single CF with the STEM package, so I believe I have a ok understanding of how the drone it self works.
But, now I trying to do some experiments with an Optitrack setup (with no additional sensors on board) and I continuously see drifting of the drone(s), even it is only the hello_world example or the individual_hover with two CFs. 

I'm doing these preliminary tests with single marker and when I look in Motive (and Rviz) or the output from mocap_helper the numbers seem correct. 
Some posts in the forum mention looking at the cf1/pose topic and when I output those number, the values for the x and y parameters continuously moves towards plus or minus 100 before returning to 0 and starting again. This happens even if the drone is stationary. 

I do have some issues with latency, but it is not all the time (I have enabled print_latency) so I would suspect that some of the position values should be forwarded correctly to the CF.

So my theory is that the values from Optitrack is not correctly mapped to the CF - is there some kind of naming scheme I need to follow in Motive? Or is there some trivial problem that I have overlooked?

**Comment by whoenig** [comment]
Sounds like you didn't update the firmware of your radio - the stock firmware doesn't support broadcast commands, i.e., the mocap data does never arrive at the drone.

**Reply by skovsen** [reply]
Thanks for the swift reply. 
I'm going back to the lab later this week and will return with my, hopefully positive, findings.
